.github/workflows/deploy-user.yml

```
@@ -5,12 +5,11 @@ on:
     branches: 
     - develop
     paths:
-      - 'application-user/**'
-      - 'application-config/**'
-      - 'common/**'
-      - 'domain-*/**'
-      - 'security-front/**'
-      - 'external-oauth/**'
+      - 'nowait-app-user-api/**'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)